homeassistant/generated/integrations.json

```
@@ -4528,6 +4528,12 @@
       "config_flow": true,
       "iot_class": "local_push"
     },
+    "plexamp": {
+      "name": "Plexamp Media Player",
+      "integration_type": "hub",
```

```suggestion
      "integration_type": "service",
```
also need to put it in the manifest

---

I'm not adding any new service (not for now at least).
I have a file that I called `service.py` where I'm putting all related to my PlexampService to have it somewhere while I work on this and that I will eventually move out.
I've been looking at what other Media Player like integrations are as integration_type and almost all of them are 'hub'. If I don't provide any service, should I still list it as 'service'?

I really appreciate your time and doing the code review. I still have it on Draft and I really wanted to know if I was very off with what I was doing, your review has given me some reassurance!